python/tvm/autotvm/tophub.py

```diff
@@ -218,12 +218,14 @@ def load_reference_log(backend, model, workload_name, template_key):
         if os.path.isfile(os.path.join(AUTOTVM_TOPHUB_ROOT_PATH, package_name)):
             find = False
             inp = None
+            counts = dict()
             for inp, res in load_from_file(filename):
+                counts[inp.target.model] = counts.get(inp.target.model, 0) + 1
                 if model == inp.target.model:
                     find = True
                     break
-            if not find and inp:
-                model = inp.target.model
+            if not find and len(counts) > 0:
+                model = max(counts.items(), key=lambda k: k[1])[0]
```

Comment:
From a conversation I had just now with Xingyu, model refers to a device type (device model, like a car model), not an ML model, and what this code is doing is finding a default device type to use in the case where there are no schedules for the actual type of the device. A comment to briefly explain this would be good.

Reply:
@broune Thanks for the advice! I will add some comments to explain context here, I agree it will help the reviewers to review the change.